apps/language_server/lib/language_server/providers/rename.ex

```diff
@@ -0,0 +1,161 @@
+defmodule ElixirLS.LanguageServer.Providers.Rename do
+  @moduledoc """
+  Provides functionality to rename a symbol inside a workspace
+
+  https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_rename
+  """
+
+  alias ElixirLS.LanguageServer.SourceFile
```

Comment:
```suggestion
  alias ElixirLS.LanguageServer.SourceFile.Path
```
And change references to just use `Path`